jbpm-wb-integration/jbpm-wb-integration-backend/src/main/java/org/jbpm/workbench/wi/backend/server/workitem/ServiceTaskHelp.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.workbench.wi.backend.server.workitem;
+
+import org.jbpm.workbench.wi.workitems.model.ServiceTaskSummary;
+
+public class ServiceTaskHelp {
+
+    public static ServiceTaskSummary createServiceTaskSummary(ServiceTask serviceTask) {
```

Comment:
Wouldn’t this make sense to be a static method in ServiceTaskSummary class itself?

Reply:
I want to use replaceall in here https://github.com/kiegroup/jbpm-wb/pull/1395/files#diff-de2b571338d9f69b537708eb8ae9427dR172.
but It will fail in GWT compile. So I create new POJO.